exercises/perfect-numbers/tests/perfect-numbers.rs

```diff
@@ -0,0 +1,35 @@
+extern crate perfect_numbers;
+
+use perfect_numbers::{Classification, classify};
+
+#[test]
+fn basic() {
+    assert_eq!(classify(0).unwrap_err(), "Number must be positive");
+}
+
+#[test]
+#[ignore]
+fn test_all() {
+    struct TestClassification {
+        num: u64,
+        result: perfect_numbers::Classification
+    }
+    let test_table: Vec<TestClassification> = vec![
+        TestClassification { num: 6, result: Classification::Perfect },
+        TestClassification { num: 28, result: Classification::Perfect },
+        TestClassification { num: 33550336, result: Classification::Perfect },
+        TestClassification { num: 33550336, result: Classification::Perfect },
+        TestClassification { num: 12, result: Classification::Abundant },
+        TestClassification { num: 30, result: Classification::Abundant },
+        TestClassification { num: 33550335, result: Classification::Abundant },
+        TestClassification { num: 2, result: Classification::Deficient },
+        TestClassification { num: 4, result: Classification::Deficient },
+        TestClassification { num: 32, result: Classification::Deficient },
+        TestClassification { num: 33550337, result: Classification::Deficient },
+        TestClassification { num: 1, result: Classification::Deficient },
+    ];
+    for t in test_table {
+        assert_eq!(classify(t.num).unwrap(), t.result);
```

Comment:
by the request of #338, I think this should be `assert_eq!(classify(t.num), Ok(t.result))` since that will display a better error message to the student if the function under test incorrectly returns an `Err`. Can you confirm whether this true, and if it is true do change it?

Reply:
Change made on the tests